go.sum

```diff
@@ -0,0 +1,15 @@
+cloud.google.com/go v0.34.0 h1:eOI3/cP2VTU6uZLDYAoic+eyzzB9YyGmJ7eIjl8rOPg=
```

Comment:
You can have multiple modules and go will automatically pick the latest version that does not conflict. The advantage to having multiple modules is that if another dependency has to scale back a version that breaks a version dependency that if you have the other version in your go.mod it shouldn't break the build.

Reply:
We discussed this offline and decided to go with one-to-one approach for now.